lib/kwtool/accounts.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Kwtool.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Kwtool.Repo
+
+  alias Kwtool.Accounts.User
```

Comment:
Let's order aliasing alphabetically.
```suggestion
  alias Kwtool.Accounts.User
  alias Kwtool.Repo
```